modules/luci-base/htdocs/luci-static/resources/validation.js

```diff
@@ -344,6 +344,12 @@ var ValidatorFactory = baseclass.extend({
 				multicast ? _('valid multicast MAC address') : _('valid MAC address'));
 		},
 
+		macwildcard: function(multicast) {
+			var m = this.value.match(/^([a-fA-F0-9]{2}|\*):([a-fA-F0-9]{2}:|\*:){4}(?:[a-fA-F0-9]{2}|\*)$/);
+			return this.assert(m != null && !(+m[1] & 1) == !multicast,
+				multicast ? _('valid multicast MAC address with wildcard (*)') : _('valid MAC address with wildcard (*)'));
+		},
+
```

Comment:
From my side, that should fit. But I remember that @jow- doesn't like to see new types being added here.

Reply:
Right, I don't like highly specific one-off types to be added here. This kind of validation above can go to a `.validate()` function directly in the form.